examples/popmon_dp_loader_example.ipynb

```diff
@@ -0,0 +1,479 @@
+    "    - Comparison of the dynamic dataloader from dataprofiler to the \n",
+    "        standard dataloader used in pandas\n",
+    "    - Popmon's usage example using both dataloaders\n",
+    "    - Dataprofiler's examples using both dataloaders\n",
+    "    - Usage of the pm_stability_report function (popmon reports)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "aec2198a",
+   "metadata": {},
+   "source": [
+    "## How to Install Popmon\n",
+    "To install popmon you can use the command below:"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4383ed2a",
+   "metadata": {},
+   "source": [
+    "`pip3 install popmon`\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "91dedc34",
+   "metadata": {},
+   "source": [
+    "From here, we can import the libararies needed for this demo."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2adec556",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import sys\n",
+    "try:\n",
+    "    sys.path.insert(0, '..')\n",
+    "    import dataprofiler as dp\n",
+    "except ImportError:\n",
+    "    import dataprofiler as dp\n",
+    "import pandas as pd\n",
+    "import popmon  # noqa"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2ed532ec",
+   "metadata": {},
+   "source": [
+    "## Comparison of Dataloaders"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cccbf4cd",
+   "metadata": {},
+   "source": [
+    "First, we have the original pandas dataloading which works for specific file types. \n",
+    "This is good for if the data format is known ahead of time but is less useful for more dynamic cases."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "96e9ff89",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def pm_dataloader(path, time_index):\n",
+    "    # Load pm dataframe (Can only read csvs unless reader option is changed)\n",
+    "    if not time_index is None:\n",
+    "        pm_data = pd.read_csv(path, parse_dates=[time_index])\n",
+    "    else:\n",
+    "        time_index = True\n",
+    "        pm_data = pd.read_csv(path)\n",
+    "    return pm_data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "16dfbe10",
+   "metadata": {},
+   "source": [
+    "Next, we have the dataprofiler's dataloader. This allows for the dynamic loading of different data formats which is super useful when the data format is not know ahead of time.\n",
+    "This is intended to be an improvement on the dataloader standardly used in pandas."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "07481259",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def dp_dataloader(path, time_index):\n",
+    "    # Datalaoder from dataprofiler used\n",
+    "    dp_data = dp.Data(path) \n",
+    "\n",
+    "    # Make dataframe from dataprofiler popmon compliant\n",
+    "    if not time_index is None:\n",
```

Comment:
can this be a list?

Reply:
Yes

Reply:
what happens if an empty list is passed?